modules/persistent-storage-using-a-shared-secret-in-a-pod.adoc

```diff
@@ -0,0 +1,57 @@
+:_content-type: PROCEDURE
+
+[id="persistent-storage-using-a-shared-secret-in-a-pod_{context}"]
+= Using a shared secret in a pod
+
+[role="_abstract"]
+To access a shared secret from a pod, you grant a given `ServiceAccount` RBAC permissions to use the `SharedSecret`.
+
+.Prerequisites
+
+* You have created a `SharedSecret` custom resource definition for the secret you wish to share across namespaces in the cluster.
+* You have user privileges to perform the following actions:
+  * Create pods and access instances of available `sharedsecrets.sharedresource.openshift.io`. In other words, you can run `oc get sharedsecrets` and get a non-empty list back.
+  * Determine if any of the service accounts available to you in your namespace are allowed to use a given `SharedSecret` custom resource. In other words, you can run `oc adm policy who-can use <identifier of specific SharedSecret>` to see if a service account in your namespace is listed.
+  * Establish, or ask someone to establish, the necessary role-based access control (RBAC) so that you can discover `SharedSecret` resources and allow service accounts to use `SharedSecret` resources.
+  * Grant the service acount of the pod permissions to use a Security Context Constraint (SCC) that allows Container Storage Interface (CSI) volumes to be mounted in a pod.
+
+.Procedure
+
+. Grant a given `ServiceAccount` RBAC permissions to use the `SharedSecret` resource in its pod:
++
+[source,terminal]
+----
+$ oc create role shared-resource-my-share --verb=use --resource=sharedsecrets.sharedresource.openshift.io --resource-name=my-share
```

Comment:
As I noted in the google doc, kubectl actually has a restriction around roles using the `use` verb.  We should note that and cite how the create the role via yaml and `oc create`

Borrow the yaml from the blog post (lining up name of the role with the name of the role cited in the role binding) or from https://github.com/openshift/csi-driver-shared-resource/blob/master/examples/simple/01-role-pod.yaml